memory/src/main/scala/filodb.memory/data/Shutdown.scala

```diff
@@ -0,0 +1,15 @@
+package filodb.memory.data
+
+import com.typesafe.scalalogging.StrictLogging
+
+object Shutdown extends StrictLogging {
+
+  def haltAndCatchFire(e: Exception, unitTest: Boolean = false): Unit = {
+    if (unitTest) {
+      throw new Error(e)
```

Comment:
Why wrap it with Error? Just throw it back out.

Reply:
I just carried over old code. You are right though. I will change.